### DISCUSSION TITLE
Best way to view Live RTMP streams on a small screen?

### DISCUSSION THREAD

**Discussion by Morphy99** [blakeblackshear/frigate#1258] [Ask A Question]
There are several ways to view the live stream but none of which seem to work well on my android device. I end up using the manufacturer app in every case which isn't ideal. The Frigate front-end seems to serve the detect streams which are low res in my case, so no good. The camera component in HA works but even though I can full screen it, I can't zoom in to the stream which I'd like to be able to do.

Is there a reason the detect stream was used in the live view within Frigate front-end?

**Comment by blakeblackshear** [comment]
The live view within frigate is rendering an mjpeg stream, and the only stream it decodes is the detect stream. Future versions may include HLS which will allow higher resolution streams in the frontend with all the same limitations (delays) as viewing the feed in HA.

**Reply by kitchung** [reply]
I'd love to see HLS with sound support!
